doc/sphinx/language/core/records.rst

```diff
@@ -88,6 +88,12 @@ Defining record types
 
    In :n:`@field_spec`:
 
+     :n:`:>` in :n:`@of_type`
+       If specified, the field is declared as a coercion from the record name
+       to the class of the field type. See :ref:`coercions`.
+       Note that this currently does nothing in :cmd:`Class` commands.
```

Comment:
That "this currently does nothing for Class" is not true, right?

Reply:
The following example shows that `>` syntax after `Class` is a noop. However the `:>` syntax actually does something. So I think this was just some confusion.
```coq
Class > B := { c : nat }.
Class A := Build_A { b :> B }.
Print HintDb typeclass_instances.
```

Reply:
Good catch, fixed.